src/FWorld.hpp

```diff
@@ -0,0 +1,35 @@
+#ifndef __FWORLD_HPP__
+#define __FWORLD_HPP__
+
+// FCharacter.cpp
+static const bool FACING_LEFT = false;
+static const float RESURRECTION_PERIOD = 3.f;
+static const float MOVEMENT_SPEED = 0.23f;
+static const float JUMPING_PERIOD = 0.4f; 
+static const FVector JUMP_VECTOR(0.f, -0.4f); // needs FVector loaded beforehand
```

Comment:
We will never be jumping to the side, only up, so this can be a `static const float JUMP_SPEED = -0.4f;` or something and the vector is assembled with `(0.f, JUMP_SPEED)` inside `FCharacter`